docs/operators/fixedWidthParserOperator.md

```diff
@@ -0,0 +1,240 @@
+
+User can set this property in `properties.xml` in the following way:
+
+```xml
+<property>
+    <name>
+      dt.application.{ApplicationName}.operator.{OperatorName}.prop.jsonSchema
+    </name>
+    <value>
+    {
+      "padding": "_",
+      "alignment": "left",
+      "fields":
+      [
+        {
+          "name": "adId",
+          "type": "Integer",
+          "length": "3",
+          "padding": "0"
+        },
+        {
+          "name": "campaignId",
+          "type": "Integer",
+          "length": "3",
+          "padding": " "
+        },
+        {
+          "name": "adName",
+          "type": "String",
+          "length": "10",
+          "alignment":"right"
+        },
+        {
+          "name": "bidPrice",
+          "type": "Double",
+          "length": "3"
+        },
+        {
+          "name": "startDate",
+          "type": "Date",
+          "format": "yyyy-MM-dd HH:mm:ss",
+          "length": "19"
+        },
+        {
+          "name": "endDate",
+          "type": "Date",
+          "format": "dd/MM/yyyy",
+          "length": "10"
+        },
+        {
+          "name": "securityCode",
+          "type": "Long",
+          "length": "5"
+        },
+        {
+          "name": "active",
+          "type": "Boolean",
+          "length": "5",
+          "trueValue": "true",
+          "falseValue": "false"
+        },
+        {
+          "name": "optimized",
+          "type": "Boolean",
+          "length": "1",
+          "trueValue": "y",
+          "falseValue": "n"
+        },
+        {
+          "name": "parentCampaign",
+          "type": "String",
+          "length": "10"
+        },
+        {
+          "name": "weatherTargeted",
+          "type": "Character",
+          "length": "1"
+        }
+      ]
+    }
+    </value>
+</property>
+```
+Where {OperatorName} is the name of the Operator and {ApplicationName} is the name of the application.
+As explained earlier padding is a character used to specify padding used in the incoming records to fill the alloted maximum space if required. User has the flexibility to specify a single padding character for the entire file or the user can choose to provide separate padding character for separate fields (columns of the record). Similar to padding character user also has the flexibility to define the alignment of the incoming records, user can choose whether the record is left,centre or right aligned.
+
+The sample json schema for a records having 3 fields as 'Occupation', 'Age' and 'Gender' having corresponding maximum space as 20,2 and 6, with corresponding padding character as '#','$' and '@' and with corresponding alignment as 'left', 'centre' and 'right' will be as follows:
```

Comment:
a records => records
fields as => fields
having corresponding maximum space as => with field widths
with corresponding padding character as => padding characters
with corresponding alignment as => and alignments